tests/pipelines/test_pipelines_auto.py

```diff
@@ -283,6 +283,26 @@ def test_from_pipe_pag_inpaint(self):
         pipe = AutoPipelineForInpainting.from_pipe(pipe_pag, enable_pag=False)
         assert pipe.__class__.__name__ == "StableDiffusionXLInpaintPipeline"
 
+    def test_from_pipe_pag_controlnet_inpaint(self):
```

Comment:
nice! we can combine this test with `test_from_pipe_pag_inpaint` and make sure it covers all the scenarios similarly as in https://github.com/huggingface/diffusers/blob/main/tests/pipelines/test_pipelines_auto.py#L141

1. `test_from_pipe_pag_inpaint` test for the inpainting XL pipeline, we can update to test SD pipeline instead because we now have a controlnet + inpaint + PAG for SD but not for SDXL
2.  make sure :
     - we test use `from_pipe` on inpainting pipeline, inpainting controlnet pipeline and pag inpainting controlnet
     - for each, make sure we test the combination of `enable_pag` and controlnet` 

but if you want to skip the test, it's ok too! we can update the test when we have the sdxl pag inpainting pipeline as well